wpilib/wpilib/spi.py

```diff
@@ -1,4 +1,4 @@
-# validated: 2015-12-30 DS 4b04073 athena/java/edu/wpi/first/wpilibj/SPI.java
```

Comment:
Sorry, didn't realise that was something that needed doing. Just read your commit about HAL stuff - I see that we need to check for that.
Test for SPI passes, but ultrasonic fail because it calls the failing `digitalsource.py` file.